lib/tasks/link_governments.rake

```diff
@@ -1,4 +1,5 @@
 namespace :governments do
+  desc "Update links for all content tagged to a government"
```

Comment:
Doesn't this also remove the government link from the content? I'm trying to figure out what line 15 is doing.

Reply:
Ah, so it's actually the opposite: we're [_only_ updating 'government' links](https://apidock.com/rails/Hash/slice). I do wonder if this might remove all the other links, but that's a problem for whoever wrote/runs this!

Reply:
Hmm, [it does appear that other links are removed](https://github.com/alphagov/publishing-api/blob/a4a054b768b7c98fcd1448f74fe6e975ab69bb2f/app/commands/v2/patch_link_set.rb) - the reference to `orphaned_content_ids` implies the other links no longer have any parents after the patch has happened 😱 I wonder if this was intended?

In any case, it looks like [this code was supposed to be deleted after it was run](https://github.com/alphagov/whitehall/pull/5183#pullrequestreview-323570282), and it [has been run now](https://trello.com/c/7yyMiJwi/1397-2-link-all-relevant-whitehall-content-to-the-appropriate-governments-in-the-publishing-api), so maybe we'd be better off deleting it. CC'ing @thomasleese.

Reply:
Yeah, I don't think we need this task anymore. We can always revert the commit that removes it if it turns out that we do.

Reply:
OK. I've added another commit to remove it, so the explanation is clearer.